### PR TITLE
feat: community links

### PR DESCRIPTION
**Pull request by rebelchris**
### Describe what this PR does
- Introducing the frontend for community links
- Flagged behind a flagsmith value
- Also introduces the new icons for scouts where applicable

### **Please make sure existing components are not breaking/affected by this PR**

**note:** Most of this is already reviewed on it's own.

### On those affected packages:
- [x] Have you done sanity checks in the webapp?
- [x] Have you done sanity checks in the extension?

### Did you test the modified components media queries?
- [x] MobileL (420px)
- [x] Tablet (656px)
- [x] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-4 #done

### CODE REVIEW

**Review by sshanzel — APPROVED** [review]
At first I was confused as it felt like I have reviewed this before. Then I realized it was an intermediate branch 😆 

Amazing to see the community links getting launched sooner and sooner! 🚢

**Review by kirillkurko — APPROVED** [review]
Looks good, left just two minor comments, but they are probably outside of this PR

File: packages/shared/src/components/BetaBadge.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React, { ReactElement } from 'react';
+import classed from '../lib/classed';
+
+const BetaBadgeContainer = classed(
+  'div',
+  'bg-theme-overlay-cabbage py-1 px-3 ml-2 rounded-10 text-theme-status-cabbage font-bold typo-body',
+);
+
+export function BetaBadge(): ReactElement {
```
[the remark this answers is not in the document]
Comment (rebelchris) [reply packages/shared/src/components/BetaBadge.tsx:9]: We do not, but thanks for raising it!
I think we generally choose named when there are multiple exports possible and default if it's only one.
(Not sure if that can be the rule?)

File: packages/shared/src/components/modals/SubmitArticleModal.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+import SubmitArticleModal from './SubmitArticleModal';
+
+const onRequestClose = jest.fn();
+
+beforeEach(async () => {
+  nock.cleanAll();
+  jest.clearAllMocks();
+});
+
+const defaultUser: LoggedUser = {
```
[the remark this answers is not in the document]
Comment (rebelchris) [reply packages/shared/src/components/modals/SubmitArticleModal.spec.tsx:20]: We can if they are all the same, but would almost think they use different user mocks, we'll have to check.
Let's add it as a seperate issue on the daily board.

File: packages/shared/src/graphql/submitArticle.ts [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+      reason
+      post {
+        id
+        title
+        commentsPermalink
+        image
+        readTime
+        numUpvotes
+        permalink
+        deleted
```
Comment (idoshamun) [inline packages/shared/src/graphql/submitArticle.ts:33]: This doesn't exists anymore as far as I remember

File: packages/shared/src/graphql/submitArticle.ts [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+  id: string;
+  url: string;
+  userId: string;
+  createdAt: Date;
+  status: string;
+  reason: string;
+}
+
+export interface SubmitArticleResposne {
+  result: 'succeed' | 'exists' | 'reject';
```
Comment (idoshamun) [inline packages/shared/src/graphql/submitArticle.ts:14]: Don't know if it's too late but it should be `rejected` if we want to keep it consistent. Minor thing

**Review by idoshamun — APPROVED** [review]
A few minor things. We're ready to go!

File: packages/shared/src/graphql/submitArticle.ts (hunk 4 above)
Comment (rebelchris) [reply packages/shared/src/graphql/submitArticle.ts:14]: Changed it in the API as well :)

File: packages/shared/src/graphql/submitArticle.ts (hunk 3 above)
Comment (rebelchris) [reply packages/shared/src/graphql/submitArticle.ts]: Removed it :)

File: packages/shared/src/components/modals/SubmitArticleModal.spec.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (rebelchris) [reply packages/shared/src/components/modals/SubmitArticleModal.spec.tsx:20]: Added a new ticket for this, I looked at it and we can indeed move it.
https://dailydotdev.atlassian.net/jira/software/projects/DD/boards/1?selectedIssue=DD-638

**Pull request merged**